BlueprintUI/Sources/Internal/LayoutTreeNode.swift

```diff
@@ -0,0 +1,72 @@
+import CoreGraphics
+import Foundation
+
+/// A node in the layout tree.
+///
+/// This tree is created during layout and measurement, and is used to hold information that may be
+/// reused if a node is visited multiple times, such as measurement caches.
+///
+/// In the layout tree, each edge is keyed by an `ElementIdentifier`, and each node corresponds to
+/// an `ElementContent`. An `ElementPath` can uniquely identify a node from the root. Elements that
+/// lazily generate subelements, such as `GeometryReader`, may create subelements with distinct
+/// identifiers across visits during a single render, which will result in spurious nodes being
+/// created that do not correspond to `LayoutResultNode` in the final layout. However, the
+/// `LayoutResultNode` tree will be an isomorphic subgraph of this tree (every `LayoutResultNode`
```

Comment:
Are you saying here that a `LayoutTreeNode` and its subgraph will be isomorphic to its `LayoutResultNode` and its subgraph, or does that `LayoutResultNode` only contain the subgraph?